docs/enterprise/docker-external-postgres.md

```diff
@@ -0,0 +1,41 @@
+# Chain Core Enterprise Edition
+
+## Connect Chain Core to an external PostgreSQL server
+This guide shows how to run Chain Core on 2 servers:
+
+1. A server running the Chain Core binary
+2. A server running PostgreSQL
+
+For this example, we will assume that server #1 runs on `10.0.0.1` and server #2 runs on `10.0.0.2`.
+
+### Setting up the PostgreSQL server
+The PostgreSQL server needs to be running PostgreSQL 9.5 and it must expose port `5432` to the server running the Chain Core binary. Here is Centos/RHEL example:
+
+```
+sudo yum -y install http://yum.postgresql.org/9.5/redhat/rhel-6.8-x86_64/pgdg-redhat95-9.5-2.noarch.rpm
+sudo yum -y install postgresql95-server postgresql95-contrib
+```
+
```

Comment:
Can probably knit these two fenced code blocks into one, with a newline in between the install and config.